Translation corrections (#200)
07th-mod/higurashi-console-arcs: Update/tsuk_005.txt

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#956f6e>圭一</color>", NULL, "<color=#956f6e>Keiichi</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 1, "ps3/s18/01/160100332", 256, TRUE);
	OutputLine(NULL, "「確かに、レナの言うとおりだな……」",

> 			NULL, "\"Yeah, good point, Rena...?\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"Yeah, good point, Rena...\"", Line_Normal);
```

//r実感が沸かない。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "実感が沸かない。",
			NULL, "It didn't feel real.", Line_Normal);
	ClearMessage();

//rその代わりに不思議と心は波立たない。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "その代わりに不思議と心は波立たない。",
			NULL, "But oddly enough, my heart didn't feel agitated.", Line_Normal);
	ClearMessage();

//r一人じゃないという安心感。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "一人じゃないという安心感。",

> 			NULL, "It was due to the relief I wasn't alone.", Line_Normal);

Reviewer: ```suggestion
			NULL, "It was the peace of mind from not being alone.", Line_Normal);
```
